build.gradle

```diff
@@ -24,6 +24,7 @@ buildscript {
 
 plugins {
     id "io.gitlab.arturbosch.detekt" version "1.0.0.RC5-6"
+    id "org.jetbrains.kotlin.kapt" version "1.2.0"
```

Comment:
If we apply kapt this way, do we also have to specify the kapt dependency, like they mention in [the docs](https://kotlinlang.org/docs/reference/kapt.html#using-in-gradle)?
```groovy
dependencies {
    kapt 'groupId:artifactId:version'
}
```

Reply:
To the best of my understanding, it seems `kapt` just replaces the usage of `annotationProcessor` in gradle. I looked for the usages before raising the PR but I couldn't find any. Please let me know if this needs to be corrected. 
Sources: [1](https://github.com/JetBrains/kotlin-examples/blob/master/gradle/android-dagger/app/build.gradle) [2](https://github.com/JetBrains/kotlin-examples/blob/master/gradle/android-databinding/app/build.gradle)

Reply:
In the two sources, they use the `apply plugin ...kapt` model so they're different than our `plugins` block.

That being said, we apply detekt in the `plugins` block without having a separate dependencies entry so this wfm.